package.json

```diff
@@ -23,10 +23,10 @@
   },
   "dependencies": {
     "hapi": "^16.3.0",
-    "hapi-confi": "^4.0.0",
+    "hapi-confi": "^4.0.1",
     "hapi-logr": "^3.2.0",
     "hapi-method-loader": "^1.3.0",
-    "hapi-route-loader": "^2.2.0",
+    "hapi-route-loader": "^2.2.1",
```

Comment:
I thought route loader was minor.  It worries me a lot if both updates were patch, that means they could get included in another release